geozero/src/gpkg/geopackage.rs

```diff
@@ -119,14 +119,18 @@ macro_rules! impl_sqlx_gpkg_decode {
 #[macro_export]
 macro_rules! impl_sqlx_gpkg_encode {
     ( $t:ty ) => {
-        impl<'q> sqlx::encode::Encode<'q, sqlx::sqlite::Sqlite> for $t {
+        impl<'q> ::sqlx::encode::Encode<'q, ::sqlx::sqlite::Sqlite> for $t {
             fn encode_by_ref(
                 &self,
-                args: &mut Vec<sqlx::sqlite::SqliteArgumentValue<'q>>,
-            ) -> std::result::Result<sqlx::encode::IsNull, Box<dyn std::error::Error + Send + Sync>>
-            {
+                args: &mut Vec<::sqlx::sqlite::SqliteArgumentValue<'q>>,
+            ) -> ::std::result::Result<
+                ::sqlx::encode::IsNull,
+                ::std::boxed::Box<
```

Comment:
god help you if you import a module named std. 🤣

Reply:
nothing wrong with OCD now and then :rofl: